assets/urls.py

```diff
@@ -0,0 +1,30 @@
+from django.urls import path, include, re_path
+from drf_yasg import openapi
+from drf_yasg.views import get_schema_view
+from rest_framework import routers, permissions
+from assets.views import AssetViewSet
+
+# Django Rest Framework Routing
+router = routers.DefaultRouter()
+router.register(r'assets', AssetViewSet)
+
+schema_view = get_schema_view(
+   openapi.Info(
+      title="Information Asset Register API",
+      default_version='v1',
+      description="University of Cambridge Information Asset Register API",
+      # terms_of_service="https://www.google.com/policies/terms/",
+      contact=openapi.Contact(email="automation@uis.cam.ac.uk"),
+      # license=openapi.License(name="BSD License"),
+   ),
+   validators=['flex', 'ssv'],
+   public=True,
+   permission_classes=(permissions.AllowAny,),
+)
+
+urlpatterns = [
+    path('', include(router.urls)),
+    path('openapi/', schema_view.with_ui('swagger', cache_timeout=None), name='schema-openapi-ui'),
```

Comment:
FWIW, the lookupproxy implementation has this under "/ui". I don't know if we want to be consistent with it?

Reply:
Also, I think the location of this page (http://localhost:8080/openapi) should be in the "getting started" doc, so a new developer can find it.

Reply:
thinking about it, we should probably have /docs/openapi/ and /docs/coreapi perhaps?

Reply:
and /docs should redirect to one of them by default

Reply:
maybe - it doesn't really matter which but we should try to be consistent. My natural inclination would be to have just one docs endpoint though unless each has something the other lacks.